src/test/java/org/fundacionjala/sfdc/stepdefinition/CommonAssertions.java

```diff
@@ -1,9 +1,11 @@
 package org.fundacionjala.sfdc.stepdefinition;
 
 import cucumber.api.java.en.Then;
+import org.fundacionjala.sfdc.core.driver.DriverManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.core.driver.DriverManager'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)